test/unit/xml-exporter/manuscript-serializer.test.ts

```diff
@@ -0,0 +1,105 @@
+import {EditorState} from "prosemirror-state";
+import {DOMSerializer} from "prosemirror-model";
+import xmldom from "xmldom";
+import {
+  createXmlDomSerializer,
+  serializeManuscript,
+  serializeManuscriptSection
+} from "../../../src/xml-exporter/manuscript-serializer";
+
+import {Manuscript} from "../../../src/model/manuscript";
+import {serializeAbstractState, serializeImpactStatementState} from "../../../src/model/abstract";
+import {serializeTitleState} from "../../../src/model/title";
+import {serializeAcknowledgementState} from '../../../src/model/acknowledgements';
+import {createBodyState, serializeBodyState} from "../../../src/model/body";
+import {serializeRelatedArticles} from "../../../src/model/related-article";
+
+jest.mock('../../../src/model/abstract');
+jest.mock('../../../src/model/title');
+jest.mock('../../../src/model/acknowledgements');
+jest.mock('../../../src/model/body', () => ({
+  serializeBodyState: jest.fn(),
+  createBodyState: jest.requireActual('../../../src/model/body').createBodyState
+}));
+jest.mock('../../../src/model/related-article');
+
+describe('serializeManuscript', () => {
```

Comment:
We should probably assert the output when passing a completely filled out manuscript and a realistic article xml entry.